web/src/app/rotations/RotationForm.js

```diff
@@ -17,7 +18,7 @@ export default class RotationForm extends React.PureComponent {
       timeZone: p.string.isRequired,
       type: p.oneOf(rotationTypes).isRequired,
       shiftLength: p.number.isRequired,
-      start: p.string.isRequired,
+      start: p.instanceOf(DateTime),
```

Comment:
This cannot be required as it can be null if the values are empty (user backspaces). Since all of the fields are required, we won't get into the instance where it breaks, as the required bit is checked before it gets to the validate func. I'll add checks on the validate functions where appropriate though as a sanity check.